src/interface/storage.thrift

```diff
@@ -142,9 +158,10 @@ struct EdgePropRequest {
     1: common.GraphSpaceID space_id,
     // partId => edges
     2: map<common.PartitionID, list<EdgeKey>>(cpp.template = "std::unordered_map") parts,
-    3: common.EdgeType edge_type,
+    3: list<common.EdgeType> edge_types,
```

Comment:
This is needed, because it is more intuitive to pass directly to the storage.

Reply:
For getNeighbors,  i think we need it.  Because it represents the over clause. And we should check the consistency with return_columns.   

But for EdgePropRequest,   do we really need it?  The users input maybe "Get props edge1.prop1, edge2.prop2".   That's much simple.

Reply:
yes, we can remove edge_types in EdgePropRequest.